pkg/bootstrap/bootstrap.go

```diff
@@ -137,10 +156,48 @@ func Stage(dataDir string, imageConf images.Images) (string, error) {
 	return refBinDir, nil
 }
 
-// extract extracts image content to targetDir all content from reader where the filename is prefixed with prefix.
-// The imageName argument is used solely for logging.
-// The destination directory is expected to be nonexistent or empty.
-func extract(imageName string, targetDir string, prefix string, reader io.Reader) error {
+// copyAll recursively copies files from source to dest, creating directories
```

Comment:
So long as we are not mutating while we copy, aka require visitor-pattern logic, then we should consider leverage https://pkg.go.dev/github.com/containerd/continuity@v0.0.0-20210208174643-50096c924a4e/fs#CopyDir which should already be in scope.

Reply:
I wish I'd known about this yesterday!